query-service/src/main/resources/configs/common/application.conf

```diff
@@ -34,6 +34,13 @@ service.config = {
       maxConnections = 5
       maxConnections = ${?POSTGRES_MAX_CONNECTIONS}
     }
+    {
+      type = trino
+      connectionString = "jdbc:trino://localhost:8080/iceberg/iceberg_gcs"
```

Comment:
Using Trino JDBC format `jdbc:trino://host:port/catalog/schema` as per https://trino.io/docs/current/client/jdbc.html#connecting

@kotharironak @arun-dhingra need to finalise the names of `catalog` and `schema`